pydoctor/themes/base/apidocs.css

```diff
@@ -114,6 +114,11 @@ li {
     padding-bottom: 5px;
 }
 
+.submodules {
```

Comment:
```suggestion
/*special css class used only in the modules index when there is more than 50 modules on the same package*/
.submodules {
```

Reply:
I updated the CSS selector to `#summaryTree .compact-modules` which is more descriptive. Is that descriptive enough or should I still add a comment? I certainly think we should avoid mentioning the exact number in the CSS, it's an implementation detail that is subject to change.